src/index.js

```diff
@@ -260,27 +262,29 @@ export default class PrismaZoom extends PureComponent {
    * @param  {MouseEvent} event Mouse event
    */
   handleMouseWheel = event => {
-    event.preventDefault()
+    if (!this.props.disableScroll) {
```

Comment:
It would be simpler to just return in the beginning of the function if disableScroll is truthy.
```
if (this.props.disableScroll) {
  return
}
/* Rest of the function */
```